ReactWindows/ReactNative/CompiledReactDelegateFactory.cs

```diff
@@ -0,0 +1,299 @@
+﻿using Newtonsoft.Json.Linq;
+using ReactNative.Reflection;
+using System;
+using System.Globalization;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using static System.FormattableString;
+
+namespace ReactNative.Bridge
+{
+    /// <summary>
+    /// A delegate factory that will compile a delegate to call the native method.
+    /// </summary>
+    public sealed class CompiledReactDelegateFactory : ReactDelegateFactoryBase
```

Comment:
This is a mistake, the file hasn't really been moved or lost history.  I'm going to delete this.  Looks like a mistaken copy of the original.